python/scrimmage/bindings/src/py_openai_env.cpp

```diff
@@ -248,51 +247,42 @@ void ScrimmageOpenAIEnv::scrimmage_memory_cleanup() {
 
 void ScrimmageOpenAIEnv::reset_scrimmage(bool enable_gui) {
     scrimmage_memory_cleanup();
-    mp_ = std::make_shared<sc::MissionParse>();
     simcontrol_ = std::make_shared<sc::SimControl>();
-    if (!mp_->parse(mission_file_)) {
+    if (not simcontrol_->init(mission_file_)) {
```

Comment:
Users of SimControl only have to create the SimControl instance, which creates its own instance of MissionParse and Log, so the user of SimControl doesn't have to create MissionParse and Log.